.github/workflows/release.yaml

```diff
@@ -0,0 +1,23 @@
+name: release
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build and copy binary
+        run: |
```

Comment:
Is it possible to utilize `make build-docker` command here?

Reply:
I tried to utilize it first and tried hard. But variable set in the `build-docker` command (`CONTAINER_ID`) couldn't be utilized in following lines.

I have tried this command on both linux and macOS, they worked but weird it does not work on CI. CI output here: https://github.com/BatuAksoy/efes/runs/8171343849?check_suite_focus=true#step:3:83

Reply:
Let me try again